libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -217,6 +219,74 @@ static void infer_completion_callback(void *args)
     task->done = 1;
 }
 
+static DNNReturnType init_model_ov(OVModel *ov_model)
+{
+    OVContext *ctx = &ov_model->ctx;
+    IEStatusCode status;
+    ie_available_devices_t a_dev;
+    ie_config_t config = {NULL, NULL, NULL};
+    char *all_dev_names = NULL;
+
+    status = ie_core_load_network(ov_model->core, ov_model->network, ctx->options.device_type, &config, &ov_model->exe_network);
+    if (status != OK) {
+        av_log(ctx, AV_LOG_ERROR, "Failed to init OpenVINO model\n");
```

Comment:
just to be specif: failed to load network,   or  ie_core_load_network returns error, or something else.

Reply:
OK, sure